docs/contributing.rst

```diff
@@ -1,84 +1,95 @@
-       | interfaces embedded into a single project.
-   * - statsd
-     - https://github.com/RackHD/on-statsd
-     - | A local statsD implementation that makes it easy to deploy
-       | on a local machine for capturing application metrics.
+We certainly welcome and encourage contributions in the form of issues and pull requests, but please read the
+guidelines in this document before you get involved.
+
+Since our project is relatively new, we don't yet have many hard and fast rules. As the project grows and more
+people get involved, we will solidify and extend our guidelines as needed.
+
+
+
+
+Understand the Repositories
+-------------------------------------------------
+
+The https://github.com/emccode/RackHD repository acts as a single source location to help you get or build all the pieces to learn about, take advantage of, and contribute to RackHD.
+
+A thorough understanding of the individual repositories is essential for contributing to the project. The repositories are described in our documentation at
+http://rackhd.readthedocs.org/en/latest/repositories.html.
+
+
+
+
+
+Submitting Contributions
+----------------------------------------
+
+
+To submit coding additions or changes for a repository, fork the repository and clone it locally. Then use a unique branch to make commits and send pull requests.
+
+Keep your pull requests limited to a single issue. Make sure that the description of the pull request is clear and complete.
+
+Run your changes against existing tests or create new ones if needed. Keep tests as simple as possible.  At a minimum, make sure your changes don’t break the existing project.
+
+After receiving the pull request, our core committers will give you feedback on your work and may request that you make further changes and resubmit the request. The core committers will handle all merges.
+
+If you have questions about the disposition of a  request, feel free to email one of our core committers:
+
+* Ben.BroderickPhillips@emc.com
+* Felix.Yuan@emc.com
+* Joseph.Longever@emc.com
+* Andrew.Hou@emc.com
+* Brian.Parry@emc.com
+* John.Frey@emc.com
+
+
+
+Issues and Bugs
+-----------------------------------
+
+Use the **Issues** section of each repository to raise issues, ask questions, and report bugs.
+
+Search existing issues to ensure that you do report a topic that has already been covered. If you have new information to share about an existing issue, add your information to the existing discussion.
+
+When reporting problems, include the following information:
+
+* Problem Description
+* Steps to Reproduce
+* Actual Results
+* Expected Results
+* Additional Information
+
+
+Coding Guidelines
+-----------------------------------
+
+Use the same coding style as the rest of the codebase. In general, write clean code and supply meaningful and comprehensive code comments.
+
+See the following sections in our documentation for guidance about a number of our standard practices:
+
+
+* `Naming Conventions`_
```

Comment:
these can be referenced with `:doc:` tags so that you don't need to maintain the links between pages manually, which may help. For example:

    * :doc:`naming-conventions`
    * :doc:`api-versioning-conventions`
    * :doc:`amqp-message-bus-conventions`